chsdi/models/vector/uvek.py

```diff
@@ -632,6 +632,37 @@ class MinergieGebaeude(Base, Vector):
 register(MinergieGebaeude.__bodId__, MinergieGebaeude)
 
 
+class StatistikwasserkraftanlagenNew(Base, Vector):
+    __tablename__ = 'statistik_wasserkraftanlagen_powerplant'
+    __table_args__ = ({'schema': 'bfe', 'autoload': False})
+    __template__ = 'templates/htmlpopup/statistikwasserkraftanlagen.mako'
+    __bodId__ = 'ch.bfe.statistik-wasserkraftanlagen'
+    __extended_info__ = True
+    __label__ = 'name'
+    id = Column('wastanumber', Integer, primary_key=True)
+    name = Column('name', Unicode)
+    location = Column('location', Unicode)
+    canton = Column('canton', Unicode)
+    hydropowerplantoperationalstatus_it = Column('hydropowerplantoperationalstatus_it', Unicode)
+    hydropowerplanttype_it = Column('hydropowerplanttype_it', Unicode)
+    hydropowerplantoperationalstatus_fr = Column('hydropowerplantoperationalstatus_fr', Unicode)
+    hydropowerplanttype_fr = Column('hydropowerplanttype_fr', Unicode)
+    hydropowerplantoperationalstatus_de = Column('hydropowerplantoperationalstatus_de', Unicode)
+    hydropowerplanttype_de = Column('hydropowerplanttype_de', Unicode)
+    beginningofoperation = Column('beginningofoperation', Integer)
+    endofoperation = Column('endofoperation', Integer)
+    dateofstatistic = Column('dateofstatistic', Unicode)
+    performanceturbinemaximum = Column('performanceturbinemaximum', Float)
+    performancegeneratormaximum = Column('performancegeneratormaximum', Float)
+    productionexpected = Column('productionexpected', Float)
+    pumpspowerinputmaximum = Column('pumpspowerinputmaximum', Float)
+    enginepowerdemand = Column('enginepowerdemand', Float)
+    fallheight = Column('fallheight', Unicode)
+    the_geom = Column(Geometry2D)
+
+register('ch.bfe.statistik-wasserkraftanlagen', StatistikwasserkraftanlagenNew)
+
+
 class STATISTIKWASSERKRAFTANLAGEN(Base, Vector):
     __tablename__ = 'statistik_wasserkraftanlagen_powerplant'
```

Comment:
@alcapat you need to remove this model.
If you wanted a smooth update you would have needed to create a new table. it's too late now and as you're defining 2 models pointing to the same table tests fail.